gto/cli.py

```diff
@@ -517,6 +517,9 @@ def register(
     bump_patch: bool = Option(
         False, "--bump-patch", is_flag=True, help="Bump patch version"
     ),
+    auto_push: bool = Option(
+        False, "--auto-push", is_flag=True, help="Push created tag automatically"
```

Comment:
Let's mark it with `experimental` for now? I anticipate some problems when remote diverges from local, so would be good to warn users we're testing and developing this functionality.
```suggestion
        False, "--auto-push", is_flag=True, help="Push created tag automatically (experimental)"
```

Reply:
good idea! I implemented the change in this https://github.com/iterative/gto/pull/281/commits/a2b0b30c0ef24dedd5cf85350186100faa3f8a65 (the change had to be done in 3 different places)